src/lib.rs

```diff
@@ -1463,6 +1463,27 @@ impl<K, V, S> Default for OrderMap<K, V, S>
     }
 }
 
+impl<K, V, S> PartialEq for OrderMap<K, V, S>
+    where K: Eq + Hash,
```

Comment:
That is a good idea!

For `K`, my first thought is `std::convert` traits and `Borrow`, but later on I found none of them are suitable here. `PartialEq` requires **symmetric**, a == b implies b == a. I'm not sure what is best for `K`, maybe they should be identical. :thinking: